Makefile

```
@@ -0,0 +1,25 @@
+COMPOSE = docker compose --env-file .env -f docker/docker-compose.yml
+
+.PHONY: up down build logs ps restart setup
+
```

checkmake (0.3.2)

[warning] Required target "all" is missing from the Makefile.

(minphony)